blackPanther.py

```diff
@@ -0,0 +1,27 @@
+from bs4 import BeautifulSoup
+import requests
+
+
+def Blackpanther_names(body):
+	#parsing
+	soup = BeautifulSoup(body.content, "html.parser")
+	soup.prettify()
+	#findall
+	#return
+	w = soup.find_all("p")	
+	count = 0
+	for c in w:
+		d = c.find_all("a")
+		for z in d:
+			l = z.find(text = True)
+			if count == 21:
+				break
+			else:
+				print(l)
+				count+=1
```

Comment:
function which scrapes data in text format.
![marvel](https://user-images.githubusercontent.com/24610214/66134378-e14afb00-e615-11e9-8421-713a1e55304b.PNG)

I have attached the output here.

Reply:
This is cool!  I wrote a version in Scrapy that I've been using but I'll merge this in as another option.  Thanks!